askametric/query_processor/query_processing_prompts.py

```diff
@@ -183,3 +247,43 @@ def create_final_answer_prompt(
     """
 
     return prompt
+
+
+def create_conversation_summary_prompt(
+    query_model: dict, context: list, context_length: int
+) -> str:
+    """Create prompt for context from previous conversation."""
+    prompt = f"""
+    Here is a question from a field employee -
+    ### Question
+    <<< {query_model["query_text"]} >>>
+
+    ===== Metadata =====
+    Here is useful metadata (might be empty if not available):
+    <<< {query_model["query_metadata"]} >>>
+
+    ===== Previous conversation =====
+    Here is a dictionary with information about the last {context_length}
+    interactions you had with the field employee (might be empty if the
+    conversation is just beginning).
+    <<<{context}>>>
+
+    ===== Instruction =====
+    I need you to do three things using the information from the previous
+    conversation:
+    1. Summarize what is relevant to the current question. Make sure to include
+    information about the relevant tables and columns. Also use the metadata to
+    construct the summary. If there is no previous conversation, simply output
```

Comment:
Why not do this programatically and avoid an api call? ie check if context is empty or does not exist as an attr and then not run this function?

Reply:
done